### DISCUSSION TITLE
Web Version

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#839] [Feature Requests]
It would be great if one could use GitJournal over the web. With Flutter's web support now in beta, it's something we should look into.

---
> Please vote on this issue with a 👍reaction. It's one of the primary mechanisms used to prioritize issues.

**Comment by MelBourbon** [comment]
Technically a web version is already available as most git repo's have a web interface. The only thing missing may be the change of the modified data and what I struggle with is the yaml format of "title" and "modified" which looks a little bit weird in for example gitea as shown below.

![image](https://user-images.githubusercontent.com/30600002/77509969-08201680-6e6e-11ea-8b7d-44d26ded5660.png)

**Comment by vHanda** [comment]
Hello. Maybe with #78, it will become easier as then for most cases a YAML header would not be required.

What is lacking for me with a Gitea/GitHub/Gitlab web versions is an easy way to browse all my notes

**Comment by MelBourbon** [comment]
> What is lacking for me with a Gitea/GitHub/Gitlab web versions is an easy way to browse all my notes

Yes, this is true.

**Comment by kozak-codes** [comment]
You could absolutely build this so it can be self hosted, or hosted in a github page like environment where everything is stored in localstorage (or using the filesystem API - https://web.dev/file-system-access/). You can also do git things in the web with isomorphic git https://isomorphic-git.org/en/ . I'd love to contribute to this if that would speed it along. Are we using a different framework for this, or is the system already built with js?
